src/editors/containers/VideoUploadEditor/index.scss

```diff
@@ -4,45 +4,45 @@
   &.active {
     border: 2px solid #262626; /* change color when active */
   }
+}
 
-  .file-upload {
-    width: 56px;
-    height: 56px;
-  }
+.pgn__dropzone {
+  height: 100vh;
+  width: 100%;
 }
 
 .video-id-container {
   width: 100%;
   justify-content: center;
 }
 
+.url-submit-button {
+  position: absolute;
+  left: 85%;
+}
+
 .video-id-prompt {
   position: absolute;
-  top: 68%;
+  top: 65%;
   left: 50%;
   transform: translateX(-50%);
-  margin-top: 1rem;
-  border: 1px solid #707070;
-  width: 308px;
-
-  input {
-    border: none !important;
-    width: 90%;
-  }
 
   input::placeholder {
     color: #454545;
+    // color: #5E35B1;
+    font-family: 'Inter';
+    font-weight: 500;
+    word-wrap: break-word;
```

Comment:
I don't think you need to specify font-family here. Inter is the font used by the [edx theme ](https://github.com/edx/brand-edx.org/blob/d4e4fa4dcd06b2fec01af97121ebd278ec95c89b/paragon/_variables.scss#L419), it is not available in the openedx theme. So if you use the edx branding variant, this font will probably already be applied, but for the the standard branding the font will be different.